build.gradle

```diff
@@ -79,3 +143,27 @@ subprojects {
   apply from: "${rootDir}/gradle/dependencies.gradle"
   apply from: "${rootDir}/gradle/java-publication.gradle"
 }
+
+task jacocoRootReport(type: JacocoReport) {
```

Comment:
Does this task run every time we `build` or must we explicitly call `jacocoRootReport`?

Reply:
yes this way it currently is in the PR is it's run as part of `build` so it will fail build if code coverage drop below the threshold, (and devs can look at the test coverage report from each build)

and `jacocoRootReport` can also be executed *independently